userland/examples/radio_rx/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <led.h>
+#include <radio.h>
+#include <timer.h>
+
+#define BUF_SIZE 60
+char packet_rx[BUF_SIZE];
+char packet_tx[BUF_SIZE];
+bool toggle = true;
+
+static void callback(__attribute__ ((unused)) int unused0,
```

Comment:
Just to add, that whatever style we use for example apps in this repository is not necessarily prescriptive for how people should build _actual_ apps. And I think we should keep that in mind for the build system, which we do generally expect people to more or less copy-paste---it should support whatever styles people want to use (which I don't think is an issue in this particular case)